### ISSUE TITLE
Location search is not working with entityService.findMany

### ISSUE THREAD

**Issue by anil1712** [notum-cz/strapi-plugin-location#193]
Hi,

Everything is working fine with inbuilt api but when I am trying to apply the filter using custom api its not filtering the records based on location provided in query params.
Here is my code snippet

const response = await strapi.entityService.findMany("api::my-custom-location.my-custom-location", {...ctx.query} );

Here is the URL

http://localhost:1337/api/restaurants?$location[coords]=49.200949303006055,16.623833585841673,5000

Thanks,
Anil

**Comment by anil1712** [comment]
@Ballonek  can you please suggest me something regarding this?

**Comment by omikulcik** [comment]
Hi @anil1712, are you sure "coords" is also name of the field you are filtering by? It was just an example you are supposed to change that based on the name of your field

**Comment by anil1712** [comment]
@omikulcik Yes I did change based on the app, but its not working, do you have any working example with Custom APIs?

**Comment by sebspi** [comment]
@omikulcik Same question from my side. How do we have to "configure" the filters when using the entity service.

```
await strapi.entityService.findMany('api::entity.entity',
      {
        filters: WHAT_TO_SET_HERE?
      }
    )
```

Can you provide an example here, please?

**Comment by fonodi** [comment]
Hi! Sorry for the delay on this, we will look at this in the following weeks.
